gtsam/slam/dataset.h

```diff
@@ -219,15 +220,21 @@ struct SfmTrack {
   SfmTrack(): p(0,0,0) {}
   Point3 p; ///< 3D position of the point
   float r, g, b; ///< RGB color of the 3D point
-  std::vector<SfmMeasurement> measurements; ///< The 2D image projections (id,(u,v))
+  std::vector<SfmMeasurement> Measurements; ///< The 2D image projections (id,(u,v))
```

Comment:
I think this should be renamed to `measurements`. The convention is to use small letters to start member variable names.